clients/drcachesim/common/options.cpp

```diff
@@ -268,8 +268,16 @@ droption_t<bytesize_t> op_warmup_refs
 (DROPTION_SCOPE_FRONTEND, "warmup_refs", 0,
  "Number of memory references to warm caches up",
  "Specifies the number of memory references to warm up caches before simulation. "
- "The warmup references come after the skipped references "
- "and before the simulated references.");
+ "The warmup references come after the skipped references and before the "
+ "simulated references. This flag is incompatible with warmup_fraction.");
+
+droption_t<double> op_warmup_fraction
```

Comment:
It would be good to add some kind of test of this option.

Reply:
Done.